JBang crashes when //DEPS line ends with space+tab

**Describe the bug**
If a JBang script's `//DEPS` directive ends with a space followed by a tab, JBang totally freaks out with a 
`[ERROR] Could not read content `

**To Reproduce**
Steps to reproduce the behavior:
1. Create a script with a dependency followed by a space and a tab, e.g.
```java
///usr/bin/env jbang "$0" "$@" ; exit $?
//DEPS com.github.lalyos:jfiglet:0.0.9 	
import com.github.lalyos.jfiglet.FigletFont;

public class hello {
  public static void main(String[] args) throws Exception {
    System.out.println(FigletFont.convertOneLine("Hello " + ((args.length>0)?args[0]:"jbang")));
  }
}
```
2. Run `jbang --verbose hello.java`
3. See error
```
jbangtest jbang --verbose hello.java
[jbang] [0:283] jbang version 0.114.0
[jbang] [0:299] Resolving resource ref: hello.java
[jbang] [0:304] Resolved resource ref as: hello.java (cached as: /Users/fbricon/jbangtest/hello.java)
[jbang] [0:333] [ERROR] Could not read content for /Users/fbricon/jbangtest
dev.jbang.cli.ExitException: Could not read content for /Users/fbricon/jbangtest
	at dev.jbang.util.Util.readFileContent(Util.java:503)
	at dev.jbang.source.Source.<init>(Source.java:55)
	at dev.jbang.source.sources.JavaSource.<init>(JavaSource.java:19)
	at dev.jbang.source.Source.forResourceRef(Source.java:139)
	at dev.jbang.source.ProjectBuilder.createSource(ProjectBuilder.java:346)
	at dev.jbang.source.ProjectBuilder.createSourceProject(ProjectBuilder.java:340)
	at dev.jbang.source.ProjectBuilder.build(ProjectBuilder.java:270)
	at dev.jbang.source.ProjectBuilder.updateProject(ProjectBuilder.java:521)
	at dev.jbang.source.ProjectBuilder.updateProjectMain(ProjectBuilder.java:476)
	at dev.jbang.source.ProjectBuilder.createSourceProject(ProjectBuilder.java:342)
	at dev.jbang.source.ProjectBuilder.build(ProjectBuilder.java:270)
	at dev.jbang.source.ProjectBuilder.build(ProjectBuilder.java:219)
	at dev.jbang.cli.Run.doCall(Run.java:73)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:145)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:21)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at dev.jbang.cli.JBang$3.handle(JBang.java:148)
	at dev.jbang.cli.JBang$3.handle(JBang.java:143)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at dev.jbang.Main.main(Main.java:14)
Caused by: java.io.IOException: Is a directory
	at java.base/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.read(FileDispatcherImpl.java:48)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:330)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:296)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:273)
	at java.base/sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:232)
	at java.base/sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:65)
	at java.base/sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:107)
	at java.base/sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:101)
	at java.base/java.nio.file.Files.read(Files.java:3244)
	at java.base/java.nio.file.Files.readAllBytes(Files.java:3295)
	at dev.jbang.util.Util.readString(Util.java:511)
	at dev.jbang.util.Util.readFileContent(Util.java:501)
	... 24 more
```

**Expected behavior**
Script should run normally

**JBang version**
>[jbang] [0:286] jbang version 0.114.0
Cache: /Users/fbricon/.jbang/cache
Config: /Users/fbricon/.jbang
Repository: /Users/fbricon/.m2/repository
0.114.0

--- a/src/main/java/dev/jbang/source/TagReader.java
+++ b/src/main/java/dev/jbang/source/TagReader.java
@@ -78,7 +78,7 @@ public abstract class TagReader {
 	}
 
 	protected Stream<String> extractDependencies(String line) {
-		return Arrays.stream(line.split(" // ")[0].split("[ ;,]+")).skip(1).map(String::trim);
+		return Arrays.stream(line.split(" // ")[0].split("[ \t;,]+")).skip(1).map(String::trim);
 	}
 
 	private static boolean isGav(String ref) {

--- a/src/test/java/dev/jbang/source/TestTagReader.java
+++ b/src/test/java/dev/jbang/source/TestTagReader.java
@@ -25,6 +25,18 @@ public class TestTagReader {
 	}
 
 	@Test
+	void testExtractDependenciesSeparator() {
+		TagReader tr = new TagReader.Extended(
+				"//DEPS foo:bar, abc:DEF:123, \thttps://github.com/jbangdev/jbang \tsomething\t ", null);
+
+		List<String> deps = tr.collectBinaryDependencies();
+		assertThat(deps, containsInAnyOrder("foo:bar", "abc:DEF:123", "https://github.com/jbangdev/jbang"));
+
+		List<String> subs = tr.collectSourceDependencies();
+		assertThat(subs, containsInAnyOrder("something"));
+	}
+
+	@Test
 	void textExtractRepositories() {
 		List<MavenRepo> repos = new TagReader.Extended("//REPOS jcenter=https://xyz.org", null).collectRepositories();
 

